walk-in-interview/src/main/webapp/job-details/script.js

```diff
@@ -0,0 +1,50 @@
+// TODO(issue/21): get the language from the browser
+const CurrentLocale = 'en';
+
+/**
+ * Import statements are static so its parameters cannot be dynamic.
+ * TODO(issue/22): figure out how to use dynamic imports
+ */
+import {AppStrings} from '../strings.en.js';
+
+import {JOB_ID_PARAM, setCookie, setErrorMessage} from '../common-functions.js';
+
+const UPDATE_JOB_PATH = '../update-job/index.html';
+
+/**
+ * Gets job Id of the current job post.
+ */
+// TODO(issue/53): implement this function
+function getJobId() {
+  return 'Not Implemented';
+}
+
+const submitButton = document.getElementById('update');
+submitButton.addEventListener('click', (_) => {
+
+  const jobId = getJobId();
+
+  if (jobId === '') {
+    setErrorMessage(/* errorMessageElementId= */'error-message',
+      /* msg= */ AppStrings['common']['empty-job-id-error-message'], /* includesDefault= */false);
```

Comment:
nit: have a const at the top for AppStrings['common'] (`const COMMON_STRINGS = AppStrings['common']`)